vignettes/datatable-reference-semantics.Rmd

```diff
@@ -258,6 +258,23 @@ flights[, c("speed", "max_speed", "max_dep_delay", "max_arr_delay") := NULL]
 head(flights)
 ```
 
+#### -- How can we update multiple existing columns in place using `.SD`?
+
+```{r}
+flights[, names(.SD) := lapply(.SD, as.factor), .SDcols = is.character]
```

Comment:
Something I can imagine will happen soon is a user trying:

```r
x[, names(.SD)[1:5] := ...]
```

Does that work already? If so, please add a test. If not, no need to handle it until it's requested later unless you see an easy fix.